src/main/java/uk/ac/ebi/eva/commons/models/converters/data/VariantStatsToDBObjectConverter.java

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright 2014-2016 EMBL - European Bioinformatics Institute
+ * Copyright 2015 OpenCB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.ac.ebi.eva.commons.models.converters.data;
+
+import com.mongodb.BasicDBObject;
+import com.mongodb.DBObject;
+import org.opencb.biodata.models.feature.Genotype;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.core.convert.converter.Converter;
+
+import uk.ac.ebi.eva.commons.models.data.VariantSourceEntry;
+import uk.ac.ebi.eva.commons.models.data.VariantStats;
+
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Converts VariantStats into MongoDb objects. Implements spring's interface of converter.
+ * <p>
+ * This class is based on OpenCGA MongoDB converters.
+ */
+public class VariantStatsToDBObjectConverter implements Converter<VariantSourceEntry, List<DBObject>> {
```

Comment:
but... it's not a PopulationStatisticsToDBObjectConverter... The PopulationStatistics is already the representation in Mongo, not the bean inside the VariantBean